### ISSUE TITLE
Improve README

### ISSUE THREAD

**Issue by nvanderperren** [VlaamseKunstcollectie/Datahub#3]
## Detailed description

The installation of this Datahub is not clearly pointed out. For example, where can I find the dev_config.yml file? _go to config_dev.yml in your datahub folder_ is not clear. It's located in  `datahub/app/config`.

## Context

It would be easier and less frustrating for everyone if the installation process is clearly explained. I think it would also be good to explain the goal of this repository, e.g. that it is used for the IIIF prototype.

## Possible implementation

A step-by-step installation guide. Could also be explained in a wiki.

## Your environment

n/a

**Comment by nvanderperren** [comment]
also adding some screenshots for the mail configuration would be handy. I can't find the _App password_ in the security settings of my google account. It's not under the _signing in to Google_ card.

**Comment by netsensei** [comment]
Good point! However, this repository is a fork of the Datahub with a few modifications for the testbed. It's primary purpose is to hold 200 "idealised" LIDO records: containing translations, values missing in registration systems,...

As such, this fork is intended as a "disposable" stub in the architecture. In a proper setup, the Imagehub components would query the production version of the Datahub. Any incomplete or inconsistent data should be fixed at the registration level.

So, instead of a step-by-step guide, let's add a "disclaimer" stating that this part is meant as a stub. 

Proper documentation should be added to the thedatahub/datahub repository.
